src/chart/gauge/GaugeSeries.ts

```diff
@@ -172,6 +172,7 @@ export interface GaugeSeriesOption extends SeriesOption<GaugeStateOption, GaugeS
 
     axisLabel?: LabelOption & {
         formatter?: LabelFormatter | string
+        rotating?: boolean
```

Comment:
I would suggest using `rotate` as the of [`series[sunburst].label.rotate`](https://echarts.apache.org/en/option.html#series-sunburst.label.rotate) and instead of using a `boolean` type, I think it would be more useful to provide the type `'tangential' | 'radial' | number`.

Reply:
> Thanks for the contribution!
> 
> Please also fix the lint problems found in the GitHub Actions.

I don‘t know why eslint config doesn't work in webstorm.
<img width="909" alt="image" src="https://user-images.githubusercontent.com/12808432/177909906-c8a4d70d-04ca-4b8e-897d-8591b2e77c39.png">


When running `npm run lint:fix` manually, nothing happen.
<img width="1075" alt="image" src="https://user-images.githubusercontent.com/12808432/177910211-1761faf6-9dbb-4138-b871-f78e9c1eff0e.png">